touchforms/formplayer/util.py

```diff
@@ -9,10 +10,23 @@ def get_xform_by_namespace(namespace):
     else:
         raise Exception("No XForm found! The database entry was " \
                         "deleted. Please syncdb and restart the server.")
-    
+
+
 def get_autocomplete_dir():
     if hasattr(settings, "TOUCHFORMS_AUTOCOMPL_DATA_DIR"):
         return settings.TOUCHFORMS_AUTOCOMPL_DATA_DIR
     else:
         root_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
         os.path.join(root_dir, 'static', 'census')
+
+
+def get_request_var(json_obj, var):
```

Comment:
we only use this for domain, and don't we now always put domain in the same part of the json body?